### PR TITLE
removed old fallbacks and hacks for Calendar style PropTypes

### PR DESCRIPTION
**Pull request by zloyrusskiy**
There was a wrong PropType fallback for react-native-web, because of bad PropType **PropTypes.shape({style: PropTypes.object})**
And there is no sense in other fallbacks, because of react-native version 0.62.2 at deps (it has typescript type checks now, not PropTypes).

P.S. fixing warnings
_index.js:1 Warning: Failed prop type: Calendar: prop type `style` is invalid; it must be a function, usually from the `prop-types` package, but received `undefined`._
and
_Attempted import error: 'ViewPropTypes' is not exported from 'react-native-web/dist/index' (imported as 'ReactNative')._

### CODE REVIEW

**Comment by Inbal-Tish** [comment]
@zloyrusskiy Hi. Please remove all formatting so I can review only the relevant code. Thanks

**Comment by zloyrusskiy** [comment]
formatting was needed because of your
image

File: src/agenda/index.js [hunk 1]
```diff
@@ -35,7 +29,7 @@ export default class AgendaView extends Component {
     /** Specify theme properties to override specific styles for calendar parts. Default = {} */
     theme: PropTypes.object,
     /** agenda container style */
-    style: viewPropTypes.style,
+    style: PropTypes.object,
```
Comment (Inbal-Tish) [inline src/agenda/index.js:32]: 'style' prop type should be PropTypes.oneOfType([PropTypes.object, PropTypes.number, PropTypes.array]) as you can pass style inline as an object or array and you can pass it from a StyleSheet where it's just a number.

File: src/agenda/index.js [hunk 2]
```diff
@@ -14,12 +14,6 @@ import {AGENDA_CALENDAR_KNOB} from '../testIDs';
 
 const HEADER_HEIGHT = 104;
 const KNOB_HEIGHT = 24;
-//Fallback for react-native-web or when RN version is < 0.44
```
Comment (Inbal-Tish) [inline src/agenda/index.js:17]: This is not a breaking change? How can I test it?

**Comment by honia19** [comment]
@zloyrusskiy should you fix comments?

File: src/agenda/index.js (hunk 1 above)
Comment (zloyrusskiy) [reply src/agenda/index.js]: ok, i have returned `View.propTypes.style` there

File: src/agenda/index.js (hunk 2 above)
Comment (zloyrusskiy) [reply src/agenda/index.js:17]: this calendar version has dependency on "react-native": "0.62.2" (it is located in devDependencies, it's wrong)
from this version till master branch there is no changes with `View.propTypes.style`

**Comment by Inbal-Tish** [comment]
@zloyrusskiy Please see merged PR #1424

**Pull request closed without merging**